apps/framework-cli/deploy/prod-load-testing/script.js

```diff
@@ -0,0 +1,30 @@
+import http from 'k6/http';
+import { check, sleep } from 'k6';
+
+// The function that defines VU logic.
+//
+// See https://grafana.com/docs/k6/latest/examples/get-started-with-k6/ to learn more
+// about authoring k6 scripts.
+//
+export default function() {
+  const url = 'http://34.82.14.129:4000/ingest/UserActivity';
+  const eventId = Math.floor(Math.random() * 1e10);
+  const userId = Math.floor(Math.random() * 1e6);
```

Comment:
Well that's a shame...

Reply:
Updated script to use placeholder values instead of random numbers.  This is a case where CodeQL correctly flagged the cryptographic insecurity but didn't realize (couldn't know without AI) that the code wasn't being used in a security context.